portal/portal-impl/impl/src/bundle/sitenav.properties

```diff
@@ -71,7 +71,7 @@ sit_othersitesclose=Close this drawer
 sit_refresh_favorites=<a href=\"javascript:location.reload()\">Reload</a> to see your updated favorite sites
 sit_star_sites_description=Star sites below to add them to your favorites bar.
 sit_star_sites_with_autofav_description=Star sites below to add them to your favorites bar. New sites will be automatically added to your favorites.
-sit_drag_sites_to_reorder=Drag sites to reorder them. Up to {0} sites can be displayed in your favorites bar.
+sit_drag_sites_to_reorder=Click and drag the <i class=\"fa fa-bars\" aria-hidden="true"></i> icon to reorder your favorites. Up to {0} sites can be displayed in your favorites bar.
```

Comment:
The double quote is escaped in the class name but un-escaped for the aria-hidden?

Reply:
Yep, copy/paste mistake. Should be good now.

Reply:
@ottenhoff, care to re-review?